web/src/components/LikertScaleQuestionField/LikertScaleQuestionField.tsx

```diff
@@ -0,0 +1,44 @@
+import { Controller, RegisterOptions, useForm } from '@redwoodjs/forms'
+import LikertScaleQuestion from '../LikertScaleQuestion/LikertScaleQuestion'
+import type { LikertScaleQuestionProps } from '../LikertScaleQuestion/LikertScaleQuestion'
+
+interface LikertScaleQuestionFieldProps extends LikertScaleQuestionProps {
+  validation?: RegisterOptions
+  errorClassName?: string
+}
+
+const LikertScaleQuestionField = (props: LikertScaleQuestionFieldProps) => {
+  const { validation, errorClassName, ...propsRest } = props
+
+  // const { className: componentClassName, style: componentStyle } =
+  //   useErrorStyles({
+  //     errorClassName: errorClassName,
+  //     name: 'likertScaleQuestion',
+  //   })
+
+  const onSubmit = (data) => {
+    console.log(data)
+  }
+
+  const methods = useForm()
+
+  return (
+    <Controller
+      control={methods.control}
+      name="likertScaleQuestion"
+      rules={validation}
+      defaultValue={[]}
+      render={({}) => (
```

Comment:
I think, for your field to work as expected, you should add a `value` and `onChange` prop to your `LikertScaleQuestion` component. That component should be "stateless" meaning you should not have a `useState` inside it. You would use the props that are supplied from the outside whenever a user interacts with your component. Does that make sens ?